docs/volto-customization/voltosettings.md

```diff
@@ -70,7 +67,7 @@ export default applyConfig;
 
 Here are some more setting you might use in your projects:
 
-- `contentIcons` - configure Content Types icons. See [documentation](https://6.docs.plone.org/volto/configuration/settings-reference.html#term-contentIcons).
+- `contentIcons` - configure Content Types icons. See [documentation](https://github.com/plone/volto/blob/main/packages/volto/src/config/index.js#L62).
```

Comment:
Use permalinks to protect against future changes.

```suggestion
- `contentIcons` - configure Content Types icons. See [documentation](https://github.com/plone/volto/blob/c72f17d3cb5002548d2c6d74b4498618b937e38a/packages/volto/src/config/index.js#L61).
```

Reply:
done